posts/2025-08-26-25.0.0.9-beta.adoc

```diff
@@ -0,0 +1,182 @@
+author_picture: https://avatars3.githubusercontent.com/navaneethsnair1
+author_github: https://github.com/navaneethsnair1
+seo-title: Support for FIPS 140-3 with IBM Semeru in 25.0.0.9-beta - OpenLiberty.io
+seo-description: Federal Information Processing Standards (FIPS) are standards and guidelines for non-military U.S. government computer systems, contractors, and agencies. In this release, you can set up Liberty to comply with the FIPS 140-3 standard when you use the IBM Semeru.
+blog_description: Federal Information Processing Standards (FIPS) are standards and guidelines for non-military U.S. government computer systems, contractors, and agencies. In this release, you can set up Liberty to comply with the FIPS 140-3 standard when you use the IBM Semeru.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Support for FIPS 140-3 with IBM Semeru in 25.0.0.9-beta
+Navaneeth S Nair <https://github.com/navaneethsnair1>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Federal Information Processing Standards (FIPS) are standards and guidelines developed by the National Institute of Standards and Technology (NIST) for non-military U.S. government computer systems, contractors, and agencies. In this release, you can set up Liberty to comply with the FIPS 140-3 standard when you use the IBM Semeru.
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to an introductory paragraph. This sentence is really
+// important because it is supposed to grab the readers attention.  Make sure to keep the blank lines
+//
+// Throughout the doc, replace 25.0.0.9-beta with the version number of Open Liberty, eg: 22.0.0.2-beta
+// // // // // // // //
+
+* <<fips, FIPS 140-3 support on IBM Semeru>>
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/32410
+// Contact/Reviewer: jimmy1wu
+// // // // // // // // 
+[#fips]
+== FIPS 140-3 support on IBM Semeru
+
+FIPS 140-3 is the latest version of the FIPS 140 standard, which provides a set of guidelines for cryptographic modules to ensure their security and integrity. Support for FIPS 140-3 was first introduced in Liberty with IBM JDK 8 in version link:https://openliberty.io/blog/2025/03/25/25.0.0.3.html[25.0.0.3]. In this release, Open Liberty supports the FIPS 140-3 standard when using link:https://developer.ibm.com/languages/semeru-runtimes[IBM Semeru Runtimes] versions 11.0.27, 17.0.15, 21.0.7, or higher. 
+
+FIPS 140-3 is enabled when:
+- security features utilizing crypto algorithms are used 
+- FIPS 140-3 Semeru jvm properties are set.
+
+If you installed the IBM Semeru Runtimes, you can enable FIPS 140-3 by setting the following system properties, for example in the `jvm.options` file:
```

Comment:
Do you need to say `for example`? Shouldn't it just be "you can enable FIPS 140-3 by setting the following system properties in the `jvm.options` file:"